### PR TITLE
Added planet learning to planetusermanual.md for Issue #590

### PR DESCRIPTION
**Pull request by kylemathias**
Did part of "manual of BeLL-Apps out of google doc #590 ". This should be good for checking off  "The PLANET Learning System / Overview"

### CODE REVIEW

File: pages/planetusermanual.md [hunk 1]
```diff
@@ -5,3 +5,30 @@
...
 
 OLE’s mission is to ensure learning for all, not only universal basic literacy and numeracy, but also vital knowledge and skills acquisition in the areas of health, nutrition, community development. Since 2007, OLE International has helped establish Open Learning Communities for children, adolescents, and young adults, including those whose lives have been disrupted by violent conflicts.  Today there are OLE affiliates in in Asia, Africa, the Middle East, and Latin America.
+
+## The PLANET Learning System
+### What is the OLE PLANET Learning System (PLANET)?
+Personalized Learning And Networked Enabled Teams (PLANET) is a, multi-lingual learning system and library with thousands of educational Resources including books, videos, and educational games.  The system also includes software tools for content and student management, student evaluation and monitoring, and Courses creation.
```
Comment (waryhermit) [inline pages/planetusermanual.md]: If you come across grammar mistakes or particular sections that need to be reworded in order to be clear, feel free to fix them up. In this case I am pretty sure that the comma after the `a` is not needed.

File: pages/planetusermanual.md [hunk 2]
```diff
@@ -5,3 +5,30 @@
...
+
+* **Nations**
+  * **Nations** always exist in the cloud and require an internet connection to access;
+  * Each **Nation** serves as a central repository for the **Community** or **a** group of communities that it serves; and
+  * Different **Nations** cannot connect to each other.
+
+* **Communities**
+  * A **Community** is a local installations of the PLANET system;
+  * A **Community** does not require an internet to access;
+  * Users connect to a **Community** via a router through a locally configured intranet using either a device with a web browser (tablet, computer, etc.);
```
Comment (waryhermit) [inline pages/planetusermanual.md]: intranet was italic and underline before... but not really sure we want that. In general it seems like an overly complicated way of saying a users can connect to the community over a LAN.. If people have thoughts on how to make this more clear I would like to hear them

File: pages/planetusermanual.md (hunk 1 above)
Comment (kylemathias) [inline pages/planetusermanual.md]: I was just copy/pasting the words. Ill be sure to look for grammar mistakes next time (all-tho i will admit i am not the best spell checker). Ill add the point you made to the Issue so others will know to look for that as well.

**Comment by kylemathias** [comment]
@dogi what did you want me to look at? What specifically?

Is there something i should do to this file? is there a reason why this hasn't been merged?

**Comment by dogi** [comment]
@kylemathias I mostly wanted to post this link so other reviewers had an easier life checking your PR

@waryhermit thanks for your acurate review which I hope @kylemathias you can fix ;)

File: pages/planetusermanual.md (hunk 2 above)
Comment (kylemathias) [inline pages/planetusermanual.md]: i guess you cant underline in markdown. underline i guess is reserved for hyperlinks.

**Comment by kylemathias** [comment]
BTW, you cant underline in Markdown. Underline is reserved for hyperlinks (Correct me if i am wrong).

**Comment by waryhermit** [comment]
@kylemathias Yeah, and underline was removed from HTLM5 as well. Its considered decoration / style and so is part of css. Its considered excessive by many people to use bold, italics and underline all together anyway.

**Pull request merged**